io/disk/ssd/blkdiscard.py.data/README

```diff
@@ -0,0 +1,5 @@
+Blkdiscard  is  used  to  discard  device  sectors.This is useful for solid-state drivers (SSDs) and thinly-provisioned storage.
+
+Values to be passed in yaml file:
+
+* Name of the device on which this script should run.
```

Comment:
80 columns limit would be great. Also, what is the actual parameter users should provide in yaml file? You don't have the yaml file to serve as example, so it would be good to be explicit here.